python/py-pydns/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pydns
+version             2.3.6
+revision            0
+distname            ${python.rootname}-${version}
+categories-append   www
+license             Python
```

Comment:
not sure if this is a correct license specification; it says "Python License (CNRI Python License)" but admittedly I am not sure either what to chose.